pkg/prometheus/agent/statefulset.go

```diff
@@ -219,6 +219,18 @@ func makeStatefulSetSpec(
 		return nil, err
 	}
 
+	var watchedDirectories []string
```

Comment:
nit: this change seems like it can be reverted, unless I'm missing something or if you believe it makes more sense have it in this part

Reply:
haha you were faster than me, I'm cleaning it up slowly :p Thanks!